factory/base.py

```diff
@@ -688,12 +688,12 @@ def _create(cls, model_class, *args, **kwargs):
         return cls._build(model_class, *args, **kwargs)
 
 
-class DictFactory(BaseDictFactory):
+class DictFactory(BaseDictFactory[T]):
```

Comment:
Likewise, this should be:
```suggestion
class DictFactory(BaseDictFactory[dict]):
```